.github/workflows/ci.yml

```diff
@@ -1608,9 +1607,10 @@ ${{ hashFiles('.pre-commit-config.yaml') }}"
           IMAGE_TAG: ${{ env.IMAGE_TAG_FOR_THE_BUILD }}
       - name: "Generate constraints"
         run: |
-          breeze generate-constraints --run-in-parallel --generate-constraints-mode source-providers
```

Comment:
longer names for "constraints-mode" but it makes sense to make them the same as the file prefixes from constraints in https://github.com/apache/airflow/tree/constraints-main